litex/soc/cores/cpu/vexiiriscv/core.py

```diff
@@ -188,6 +188,17 @@ def args_read(args):
         VexiiRiscv.no_netlist_cache = args.no_netlist_cache
         VexiiRiscv.vexii_args      += " " + args.vexii_args
 
+        if "--with-rva" in VexiiRiscv.vexii_args:
+            VexiiRiscv.with_rva = True
+        if "--with-rvc" in VexiiRiscv.vexii_args:
+            VexiiRiscv.with_rvc = True
+        if "--with-rvf" in VexiiRiscv.vexii_args:
+            VexiiRiscv.with_rvf = True
+        if "--with-rvd" in VexiiRiscv.vexii_args:
+            VexiiRiscv.with_rvd = True
+        if "--with-mul" in VexiiRiscv.vexii_args and "--with-div" in VexiiRiscv.vexii_args:
```

Comment:
Hi ^^
if ("--with-mul" in VexiiRiscv.vexii_args and "--with-div" in VexiiRiscv.vexii_args) or  "--with-rvm" in VexiiRiscv.vexii_args:

:)